src/mon/MgrMap.h

```diff
@@ -179,6 +179,17 @@ class MgrMap
     return false;
   }
 
+  bool enabled_module(const std::string &module_name) const
```

Comment:
Just wondering: would it make sense to rename this to `module_enabled` or `is_enabled`?

Reply:
updated.